### PR TITLE
Fix detected defects

### PR DESCRIPTION
**Pull request by jvc931**
Fix detected defects. The video with the application are on this link https://drive.google.com/open?id=0B4gotoWSQOXcYzVwN2tycTJMbVU

### CODE REVIEW

**Review by daniellemariani — COMMENTED** [review]
Provide a video of these fixes

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -20,6 +20,7 @@
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
-        <activity android:name=".presentation.view.activity.HomeActivity"></activity>
+        <activity android:name=".presentation.view.activity.HomeActivity"
+            android:configChanges="keyboard|keyboardHidden|orientation|screenLayout|uiMode|screenSize|smallestScreenSize"></activity>
```
Comment (daniellemariani) [inline app/src/main/AndroidManifest.xml]: why do you need *uiMode*?

Comment (jvc931) [reply app/src/main/AndroidManifest.xml]: No, I read better and only need orientation and screenSize, I will make a commit to fix that.

**Review by daniellemariani — APPROVED** [review]
LG

**Pull request merged**